---
layout: compress
---

<!DOCTYPE html>
<html>

<head>
    {% include head.html %}
</head>

<body>
    {% include navbar.html %}

    <!--Main Section-->
    <section class="hero is-fullheight" id="project">
        <div class="hero-body">
            <div class="container has-text-left has-text-black-bis">
                <!-- <div class="section">
                    <p class="subtitle is-uppercase has-text-weight-medium has-text-grey">Projects</p>
                </div> -->
                <!-- {% include project-card.html %} -->
                <br><br>
                <p style="text-align: center;">The complete list of publications is available on
                    <a href="https://scholar.google.com/citations?user=pNfbA7EAAAAJ">Google Scholar</a>.
                </p>
                <br>
                <div class="subtitle">
                    Methods Development
                </div>
                <p>We develop both computational (<a href="https://www.science.org/doi/10.1126/science.aan3235">Karaiskos, Wahle et al., 2017, Science</a>;
                    <a href="https://www.nature.com/articles/s41586-019-1773-3">Nitzan, Karaiskos et al., 2019, Nature</a>;
                    <a href="https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giac064/6646447">Sztanka-Toth et al., 2022, Gigascience</a>;
                    <a href="https://academic.oup.com/nargab/article/4/2/lqac042/6603620?login=true">Senel et al., 2022, NAR Genomics and Bioinformatics</a>)
                and experimental
                    (<a href="https://www.nature.com/articles/s41592-019-0503-y">Legnini, Alles et al., 2019, Nature Methods</a>;
                    <a href="https://www.nature.com/articles/s41592-023-01986-w">Legnini et al., 2023, Nature Methods</a>)
                    methods to study tissues and answer complex biological questions.
                </p>
                <br>
                <p>Recently, we developed an inexpensive, do-it-yourself, high-resolution, and open-source
                    method to quantify gene expression directly in tissues (Schott, Leon-Perinān, Splendiani et al. 
                    "Open-ST: High-resolution spatial transcriptomics in 3D" 
                    <a href="https://www.cell.com/cell/abstract/S0092-8674(24)00636-6">2024, Cell</a>;
                    <a href="https://www.sciencedirect.com/science/article/pii/S2666166724006865">2024, STAR Protocols</a>;
                    <a href="https://rajewsky-lab.github.io/openst/latest/">Github</a>).
                    </p>
                </br></br>
                <div class="subtitle">
                    Clinical Applications
                </div> 
                <p>We are actively collaborating with clinicians from the Charité University and other
                    clinics (e.g. Kidney: <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-022-01108-9">Hinze et al.,
                        2022, Genome Medicine</a>,
                        Urine: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0085-2538(22)00688-3">Klocke et al.,
                            2022, Kidney International</a>,
                            Lung: <a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(25)00094-8">Pentimalli et al.,
                                2025, Cell Systems</a>).
                </p>
                <br>
                <p>
                    We constructed the first high-resolution molecular tumor atlas in 3D 
                    (from a single patient) and learned from these ~1 million sequenced cells 
                    (in tissue space) which pathways and gene programs drive phenotypes of the 
                    primary and metastatic tumor.
                </p>
                <br>
                <p>
                    We currently routinely generate spatial transcriptomics data as a highly 
                    informative readout for how cells communicate within tissues/tumors and how 
                    they react to perturbations (disease mutations, perturbations from the 
                    environment, viral infection, microplastics, etc). 
                </p>
                 </br></br>
                <div class="subtitle">
                    Organoids
                </div>
                <p>
                    In a human brain organoid model, we found that Herpes induced neural damage 
                    after Herpes activation (a debilitating clinical problem and practically 
                    impossible to study in patients) can be prevented in brain organoids by 
                    treating them not only with acyclovir (the standard drug given in the clinics) 
                    but also with inflammation blockers 
                    <a href="https://www.nature.com/articles/s41564-023-01405-y">Rybak-Wolf et al., 2023, Nature Microbiology</a>. 
                </p>
                <br>
                <p>
                    We also invented an optogenetic approach to perturb expression of specific 
                    genes at precisely located regions/cells within organoids 
                    <a href="https://www.nature.com/articles/s41592-023-01986-w">Legnini et al., 2023, Nature Methods</a>.
                    This approach allows, for example, to work with "programmed
                    organoids" to test specific questions about how gene interactions organize
                    tissue function in health and disease.
                </p>
                </br></br>
                <div class="subtitle">
                    Gene Regulation
                </div> 
                 <p>We showed that the interaction between microRNA miR-7 and the circular 
                    RNA CDR1as 
                    (<a href="https://www.nature.com/articles/nature11928">Memczak et al., 2013, Nature</a>;
                    <a href="https://www.science.org/doi/10.1126/science.aam8526">Piwecka et al., 2017, Science</a>>) 
                    is regulating glutamatergic transmission and neural connectivity 
                    (<a href="https://www.embopress.org/doi/full/10.1038/s44319-024-00168-9">Cerda Jara et al., 2024, EMBO Reports</a>).
                    </p>
                    <br>
                    <p>
                    We could also show that miRNA evolution in vertebrates and the octopus share common global
                    features 
                    (<a href="https://www.science.org/doi/10.1126/sciadv.add9938">Zolotarov et al., 2022, Science Advances</a>)
                    and, excited by these insights, are now studying the function of primate specific miRNAs in 
                    human brain development.</p>
            </div>
        </div>
    </section>
    </div>

    <!--Footer-->
    {% include footer.html %}
</body>

</html>